open-company/open-company-web
# Secure post restyle.

Card: https://trello.com/c/OETtj4dA

Abstract: https://share.goabstract.com/f8ae72a8-a0e2-42cd-b331-18c47819e0df

To test:
- share a post
- select URL (if you staging select Public read only)
- copy the url
- open an incognito window (ie: not logged in)
- navigate to the url
- [x] can you see the top dark header asking to login? Good
- [x] can you remove the header clicking the X? Good
- now go to login
- login or signup with a different account/team/org
- nav to url
- [x] can you NOT see the top dark header asking to login? Good
